tests/run-make/num-fmt/rmake.rs

```diff
@@ -0,0 +1,21 @@
+// Test to ensure that if you use a `NumBuffer` with a too small integer, it
+// will fail at compilation time.
+
+// FIXME: This test should be a `ui` test, but not possible for the following reasons:
+// * With the default `--emit`, the compilation succeeds, we can go around by using `--emit=link`
+//   but then we get the second problem.
+// * Compilation error is not generated in the JSON output.
+//
+// So for not we go around this problem by moving it in `run-make` until we figure out what's
+// going on.
```

Comment:
Maybe you need `//@ build-fail` + `//@ ignore-pass`?

Reply:
Oh, does that const-eval panic not have span info? Let me double-check this locally

Reply:
Maybe not indeed. How can we provide the missing span info?